modules/ROOT/pages/vm/vm-publish-listen.adoc

```diff
@@ -1,25 +1,27 @@
-= Example: To Publish and Listen to Messages through the VM Connector - Mule 4
+= Example: Publish and Listen to Messages through the VM Connector - Mule 4
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 
-This example shows how to publish a message that is received by a different flow.
+The following example shows how to publish a message that is received by a different flow. To do this you use the `<vm:publish>` and `<vm:listener>` elements that point to the same queue.
+
+. Define a flow named `listener` that receives the published message from the subsequent `publish` flow. The `listener` flow contains a `<vm:listener>` as a message source and points to `queueName="myQueue"`, for example:
 
-. Define a flow with a `<vm:listener>` as a message source, for example:
-+
 [source,xml,linenums]
 ----
 <flow name="listener">
    <vm:listener queueName="myQueue" config-ref="vm"/>
    <flow-ref name="yourIntegrationLogic"/>
 </flow>
 ----
-+
-. Feed messages into the newly created queue.
-+
-Suppose you have a flow that is triggered by an HTTP request with a JSON payload and you want to extract the `body` attribute of that JSON object and publish it to the queue, for example:
-+
+
+[start=2]
+. Define a flow named `publish` that references to `myQueue` queue that you created in the `listener` flow. The `publish` flow is triggered by an HTTP request with a JSON payload where you want to extract the `body` attribute of that JSON object and publish it to the queue created in the `listener` flow.
+
+* The `publish` operation is one-way. Once the message is posted into the queue, the flow continues. The processing on the listening side happen asynchronously. You do not receive notification of its outcome.
+* The `content` parameter allows you to use DataWeave to build the value to send (without producing side effects on the message). If you do not specify a `content`, the entire payload will be sent.
```

Comment:
Change "allows you to" to "enables you to" (Kim always marks this one on my stuff).

Suggested wording - I don't really understand what is meant by producing side effects on the message, so I removed it.

"The `content` parameter enables you to use DataWeave to build the value to send. If you do not specify a `content`, the entire payload is sent."